runtime/Cpp/runtime/src/atn/ATNConfigSet.cpp

```diff
@@ -208,7 +208,7 @@ std::string ATNConfigSet::toString() const {
     ss << ",uniqueAlt = " << uniqueAlt;
   }
 
-  if (conflictingAlts.size() > 0) {
+  if (conflictingAlts.count() > 0) {
```

Comment:
Is this a container that has empty() ? Then this is typically the cheapest implementation for that question.

`if (!conflictingAlts.empty()) { ... }`